hotels.html

```diff
@@ -57,45 +56,59 @@ <h2 class="visually-hidden">Список жилья</h2>
           <a href="#">
             <h3 class="catalog-item-title">Amara Resort & Spa</h3>
             <p class="catalog-item-image">
-              <img src="https://via.placeholder.com/135x90" width="135" height="90" alt="Amara Resort & Spa">
+              <img src="img/Amara Resort & Spa.jpg" width="135" height="90" alt="">
```

Comment:
зря убрал alt

 и имя файла с пробелом - может вылится в проблемы, переименуй файл
```suggestion
              <img src="img/amara-resort-spa.jpg" width="135" height="90" alt="Amara Resort & Spa">
```

или приходи на консультацию